api/about.json

```diff
@@ -0,0 +1,9 @@
+{
+ "0.0.1": [
+   {
+    "logo": "http://bit.ly/2niBg5Y",
+    "website":"https://pydelhi.org/",
+    "details":"PyDelhi is a volunteer organization, group of technically inclined students, professionals, teachers and entrepreneurs who want to learn something new having a common primary interest in pyhton." 
```

Comment:
Spelling of Python is wrong :(

@akshayaurora please wait before merging. :(

I'm fixing it now.